tests/load/pipeline/test_arrow_loading.py

```diff
@@ -296,3 +300,57 @@ def some_data():
     # Load is successful
     info = pipeline.load()
     assert_load_info(info)
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True, default_staging_configs=True, subset=["duckdb"]),
+    ids=lambda x: x.name,
+)
+@pytest.mark.parametrize("is_none", [True, False])
+def test_warning_from_arrow_normalizer_on_null_column(
```

Comment:
this is also a good place to run arrow tests. another good place is `test_pipeline_extra`. those tests we also run on mac on windows